cli/azd/pkg/executil/executil.go

```diff
@@ -197,10 +198,31 @@ type RunArgs struct {
 	EnrichError bool
 }
 
+type redactData struct {
+	matchString   *regexp.Regexp
+	replaceString string
+}
+
 func (rr RunResult) String() string {
 	return fmt.Sprintf("exit code: %d, stdout: %s, stderr: %s", rr.ExitCode, rr.Stdout, rr.Stderr)
 }
 
+func redactSensitiveData(msg string) string {
+	regexpRedactRules := make(map[string]redactData)
```

Comment:
You should be able to use a map literal here (instead of doing a make + an assign).  See https://go.dev/tour/moretypes/20 for an example.